dist/scripts/update-firmware-binaries.js

```diff
@@ -163,26 +167,25 @@ var updateSettings = function updateSettings() {
     versionNumber = versionNumber.substr(1);
   }
 
-  var settings = (0, _stringify2.default)((0, _extends3.default)({
+  var scriptSettings = (0, _stringify2.default)((0, _extends3.default)({
     versionNumber: versionNumber
   }, DEFAULT_SETTINGS), null, 2);
+
   var settingsBinaries = [];
-  settings = settings.replace(/(system-part\d-).*(-.*.bin)/g, function (filename, part, device) {
+  scriptSettings = scriptSettings.replace(/(system-part\d-).*(-.*.bin)/g, function (filename, part, device) {
```

Comment:
not sure how this should be named..it was settings and it was shadowing with imported app settings.